### PR TITLE
Update list of topics and add meetup and presentations so far

### PR DESCRIPTION
**Pull request by adrianmo**

### CODE REVIEW

**Comment by adrianmo** [comment]
This adds Jordi Anguela talk and closes #4

File: TOPICS.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
++ Testing libraries and frameworks
+  * Any core library
+  * Behat
+  * Capybara
+  * Cucumber
+  * Gospec
+  * Jasmine
+  * Junit
+  * Mocha
+  * Phpspec
```
Comment (jpahullo) [inline TOPICS.md:56]: Could you add "PHPUnit" please?

File: readme.md [hunk 2]
```diff
@@ -2,20 +2,37 @@
...
+  * [How to leverage Docker thanks to Kubernetes](https://pipo02mix.github.io/slides/git-workflows.html#/) by *[Fernando Ripoll](https://github.com/pipo02mix)*
 
-For the moment, this is the information available here:
 
-* [Meetup events](http://www.meetup.com/Tarragona-Developers-Meetup/)
-* [Proposals for next meetups](https://github.com/tgndevs/meetup/blob/master/meetups.md)
-* [Slides](https://github.com/tgndevs/slides)
+## List of topics
+
+If you are interested in presenting at the Tarragona Developers Meetup but you are not sure what topics can fit the meetup or need some ideas please check out our [TOPICS.md](TOPICS.md) file.
```
Comment (jpahullo) [inline readme.md:33]: being the same, I think it is equivalent to `[[TOPICS.md]]`.

**Comment by jpahullo** [comment]
@adrianmo, it is a major improvement! Wow! What was just an starting point, it is becoming an awesome project.

There is a couple of comments that, if you please, you could address very quick. Thanks in advance!

**Comment by adrianmo** [comment]
@jpahullo, I've just added PHPUnit. The "List of Topics" section on README.md is only to give visibility to the TOPICS.md file and provide a link to it.

**Comment by jpahullo** [comment]
Awesome!

**Pull request merged**